Hackney.Shared.CautionaryAlerts/Factories/EntityFactory.cs

```diff
@@ -120,7 +120,7 @@ public static PropertyAlertNew ToDatabase(this CautionaryAlertListItem entity)
             };
         }
 
-        public static PropertyAlertNew ToDatabase(this CreateCautionaryAlert entity)
+        public static PropertyAlertNew ToDatabase(this CreateCautionaryAlert entity, string AlertId, bool IsActive)
```

Comment:
This is the C# parameter naming convention violation.
Parameters should be camelCased instead of PascalCased.